plugins/vpp/l3plugin/data_resync.go

```diff
@@ -137,10 +141,46 @@ func (plugin *RouteConfigurator) Resync(nbRoutes []*l3.StaticRoutes_Route) error
 			}
 		}
 	}
+
+	// Remove other routes except DROP type
+	if len(vppRouteDetails) > 0 {
```

Comment:
This check seems useless. It doesn't affect anything, just adds extra indentation.

Reply:
already removed